packages/vue-renderer/src/renderer.js

```diff
@@ -413,6 +413,7 @@ export default class VueRenderer {
       const hash = crypto.createHash(hashAlgorithm)
       hash.update(serializedSession)
       cspScriptSrcHashes.push(`'${hashAlgorithm}-${hash.digest('base64')}'`)
+      HEAD += cspScriptSrcHashes.join()
```

Comment:
I think this would be better to added after `vue-renderer:ssr:csp` hook. Also shouldn't we wrap it around any tag?